src/ZoneServer/Buffs/Handlers/Archers/Wugushi/LatentVenom_Debuff.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using Melia.Shared.Game.Const;
+using Melia.Zone.Buffs.Base;
+using Melia.Zone.Network;
+using Melia.Zone.Skills.Combat;
+using Melia.Zone.World.Actors;
+using static Melia.Zone.Skills.SkillUseFunctions;
+
+namespace Melia.Zone.Buffs.Handlers.Archers.Wugushi
+{
+	/// <summary>
+	/// Handle for the Latent Venom Debuff, which ticks damage every second.
+	/// </summary>
+	[BuffHandler(BuffId.LatentVenom_Debuff)]
+	public class LatentVenom_Debuff : BuffHandler
+	{
+		public override void OnStart(Buff buff)
+		{
+			var damageThickDelay = buff.Data.UpdateTime;
+
+			Crescendo_Bane_Buff.TryApply(buff.Caster, ref damageThickDelay);
+
+			buff.UpdateTime = damageThickDelay;
+		}
+
+		public override void WhileActive(Buff buff)
+		{
+			if (buff.Target.IsDead)
+				return;
+
+			if (!buff.Caster.TryGetSkill((SkillId)buff.NumArg2, out var skill))
+				return;
+
+			var damageMultiplier = 1f;
```

Comment:
I think it's ok the way it was implemented. I tried to pass the damage through the arg1/2 but it will be static so we can't do that. And a static handler for damaging seems very weird, each buff has it's own specification.

Reply:
I didn't mean for damaging, but damage calculation, which the skill handlers need to do anyway. If something about the skill calculations changes we ideally don't want to also have to copy these changes to a bunch of buff handlers, assuming that we even remember that those exist.

Reply:
it will look ugly but I will do it.